lib/setup/setupSetup.js

```diff
@@ -182,7 +182,7 @@ function Setup(options) {
             if (!otype) {
                 otype = 'file';
             } else {
-                otype = (otype || '').toLowerCase();

```

Comment:
Unnecessary, we just checked it 3 lines above

Reply:
ok